src/js/modules/liked-home.js

```diff
@@ -0,0 +1,9 @@
+import likedIconimg from '../../img/heart-icon-liked.png';
```

Comment:
I'll open an issue to try to figure out how to keep all the resource-loading code in index.js since it tends to cause problems to the testing environment. Or to figure out how to configure jest to deal with this.

Reply:
Great :+1: